config/session.js

```diff
@@ -10,44 +10,49 @@
-   *                                                                          *
-   * Customize when built-in session support will be skipped.                 *
-   *                                                                          *
-   * (Useful for performance tuning; particularly to avoid wasting cycles on  *
-   * session management when responding to simple requests for static assets, *
-   * like images or stylesheets.)                                             *
-   *                                                                          *
-   * https://sailsjs.com/config/session                                       *
-   *                                                                          *
-   ***************************************************************************/
-  //// Sails Default
-  // isSessionDisabled: function (req){
-  //   return !!req.path.match(req._sails.LOOKS_LIKE_ASSET_RX);
-  // },
+   /***************************************************************************
+    *                                                                          *
+    * Customize when built-in session support will be skipped.                 *
+    *                                                                          *
+    * (Useful for performance tuning; particularly to avoid wasting cycles on  *
+    * session management when responding to simple requests for static assets, *
+    * like images or stylesheets.)                                             *
+    *                                                                          *
+    * https://sailsjs.com/config/session                                       *
+    *                                                                          *
+    ***************************************************************************/
+   //// Sails Default
+   // isSessionDisabled: function (req){
+   //   return !!req.path.match(req._sails.LOOKS_LIKE_ASSET_RX);
+   // },
 
-  //// Don't disable sessions for "/plugin/*"
-  // isSessionDisabled: function (req) {
-  //   // Enable session for all plugin requests.
-  //   if (req.path.match(/^\/plugin\//)) { 
-  //     return false;
-  //   }
-  //   // Otherwise, disable session for all requests that look like assets.
-  //   return !!req.path.match(req._sails.LOOKS_LIKE_ASSET_RX);
-  // },
+   //// Don't disable sessions for "/plugin/*"
+   // isSessionDisabled: function (req) {
+   //   // Enable session for all plugin requests.
+   //   if (req.path.match(/^\/plugin\//)) {
+   //     return false;
+   //   }
+   //   // Otherwise, disable session for all requests that look like assets.
+   //   return !!req.path.match(req._sails.LOOKS_LIKE_ASSET_RX);
+   // },
 
-  //// Never disable sessions
-  isSessionDisabled: function (req) {
-    return false;
-  },
+   //// Never disable sessions
+   isSessionDisabled: function (/* req */) {
+      return false;
+   },
 
+   // Use redis in swarm for session store
```

Comment:
can I just say how happy I am that we are finally switching sails over to redis storage?  This opens up the way for us to scale our api on the swarm.